modules/tinymce/src/plugins/imagetools/main/ts/core/Actions.ts

```diff
@@ -232,7 +232,7 @@ const flip = function (editor: Editor, imageUploadTimerState, axis) {
 };
 
 const handleDialogBlob = function (editor: Editor, imageUploadTimerState, img, originalSize, blob: Blob) {
-  return new Promise(function (resolve) {
+  return new Promise(function (_resolve) {
```

Comment:
This seems very odd to me. Why are we creating a promise that we never resolve?

Reply:
I'm guessing this is a refactoring gone bad... we never do anything with the returned promise and we actually already have a promise inside this function, so we probably just need to remove creating the new promise and return the inner promise I guess?

Reply:
you definitely found a code smell, this could be turned into a simple setTimeout since the return value can (and probably should) be void

Reply:
I removed the wrapping promise.